spring-cloud-skipper-docs/src/main/asciidoc/appendix-building.adoc

```diff
@@ -28,21 +28,208 @@ We try to cover this in the `.mvn` configuration, so, if you find you have to in
 To generate only the REST Docs documentation, use the following command:
 
 ----
-./mvnw test -pl spring-cloud-skipper-server-core -Dtest=*Documentation*
+$ ./mvnw test -pl spring-cloud-skipper-server-core -Dtest=*Documentation*
 ----
 
 To build the only the Asciidoctor documentation, use the following command:
 
 ----
-./mvnw package -DskipTests -Pfull -pl spring-cloud-skipper-docs
+$ ./mvnw package -DskipTests -Pfull -pl spring-cloud-skipper-docs
+----
+
+=== Custom Server Build
+This chapter contains instructions how to create a custom server
+build and should cause exactly same packaged _uber-jar_ compared to
+one from a Skipper build itself.
+
+It is required to follow same _Spring Boot_ main class structure used in Skipper itself.
+Example of it is shown below:
+
+[source,java,indent=0,subs="attributes+"]
+----
+package com.example.customskipperserver;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.actuate.autoconfigure.ManagementWebSecurityAutoConfiguration;
+import org.springframework.boot.autoconfigure.security.SecurityAutoConfiguration;
+import org.springframework.boot.autoconfigure.session.SessionAutoConfiguration;
+import org.springframework.cloud.deployer.spi.cloudfoundry.CloudFoundryDeployerAutoConfiguration;
+import org.springframework.cloud.deployer.spi.kubernetes.KubernetesAutoConfiguration;
+import org.springframework.cloud.deployer.spi.local.LocalDeployerAutoConfiguration;
+import org.springframework.cloud.skipper.server.EnableSkipperServer;
+
+@SpringBootApplication(exclude = {
+                CloudFoundryDeployerAutoConfiguration.class,
```

Comment:
Can we add a note on why we exclude these auto configuration classes?

Reply:
I wonder what should we write other than all these auto-configs simply break skipper if not excluded?

Reply:
Actually we have #561 which is a bit broader issue around this fact that these needs to be excluded.